tools/target-browser/backup.go

```diff
@@ -83,7 +83,7 @@ func (auth *AuthInfo) TriggerAPI(pathParam, queryParam, apiPath string, selector
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK || resp.Body == nil {
+	if resp.StatusCode != http.StatusOK && resp.Body == nil {
```

Comment:
why this change?
Just test these changes once, if  `StatusCode` is OK and `resp.Body` is nil, all commands are working properly and printing of response is not giving error for all formats.

Reply:
Reverted